src/main/java/cc/hyperium/integrations/betterchatfilter/BetterChatFilter.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *     Copyright (C) 2018  Hyperium <https://hyperium.cc/>
+ *
+ *     This program is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU Lesser General Public License as published
+ *     by the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     This program is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU Lesser General Public License for more details.
+ *
+ *     You should have received a copy of the GNU Lesser General Public License
+ *     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+// Created by RDIL in November 2018.  
+// Created in hope to limit toxicity.  
+
+package cc.hyperium.integrations.betterchatfilter;
+import cc.hyperium.mods.sk1ercommon.Multithreading;
+import cc.hyperium.Hyperium;
+import cc.hyperium.event.EventBus;
+import cc.hyperium.handlers.handlers.HypixelDetector;
+import cc.hyperium.config.Settings;
+import cc.hyperium.event.ChatEvent;
+import cc.hyperium.event.InvokeEvent;
+import net.minecraft.client.Minecraft;
+import cc.hyperium.utils.ChatColor;
+import java.lang.*;
+import java.lang.System;
+import java.io.*;
+import java.util.*;
```

Comment:
wildcards >:((((((
google how to disable them

Reply:
I disagree with you here asbyth, nothing wrong with a wildcard imo.